Feature/section destinations

lexanikitin opened the pull request:
Ветка "Наши направления"

lexanikitin committed:
feat: карточки в секции, с адаптивной верстке

lexanikitin committed:
refactor: убрал лишние сущности

lexanikitin committed:
Merge branch 'develop' into feature/section-destinations

lexanikitin committed:
Merge remote-tracking branch 'origin/develop' into feature/section-destinations

lexanikitin committed:
feat: кнопка - ссылка, адаптив верстки

lexanikitin committed:
Merge branch 'develop' into feature/section-destinations

lexanikitin committed:
bugfix: правка путей к картинкам

lexanikitin committed:
feat: порядок карточек соответствует макету на всех устройствах

SergeyKukhtalevV commented:
1. Если ссылки ведут на другие страницы, то необходимо прописывать атрибут target 
2. В промежуточных состояниях (между десктопом, планшетом и смартфоном) заголовок уползает влево. Думаю, для более корректного адаптива попробовать зафиксировать позицию title на одной вертикальной линии с левой гранью карточек.
В остальном все ок.

SashkaStudent reviewed (commented):
1. Можно сделать переносы слов заголовков как в макете, используя неразрывный пробел (&nbsp)
![nbsp](https://user-images.githubusercontent.com/40184171/184461060-e7d99b95-40fc-4cd0-bad7-f9b069534bee.gif)
2. У карточек .activities__card должен меняться border-radius на разных устройствах
3. Кнопки для мобайл и таблет имеют не тот размер
 
![br-and-linksize](https://user-images.githubusercontent.com/40184171/184461566-a2850a08-1813-462b-a52a-aeccfbe73ea5.gif)
![br-and-linksize-mobile](https://user-images.githubusercontent.com/40184171/184462763-72e3e82d-1cf9-45d7-98fc-c4e91e471085.gif)

4. Обращаю внимание, что между разрешениями 670px - 800px происходит перескок количества колонок (если изменять ширину окна браузера).
![jumping](https://user-images.githubusercontent.com/40184171/184463097-55a884c7-91aa-4512-95b3-a55c96ac363e.gif)
5. Пустой класс в activities__cards

lexanikitin committed:
bugfix: target для ссылок

lexanikitin committed:
bugfix: улучшение в типографике контента

lexanikitin committed:
bugfix: убрал пустой класс

lexanikitin committed:
bugfix: исправления в верстке:

исправлен радиус скругления углов карточек для носимых устройств,
осправлено позциорирование карточке в переходных разрешения между моб и планшетной вертской.

SergeyKukhtalevV reviewed (approved):
ок

lexanikitin committed:
Merge branch 'develop' into feature/section-destinations

# Conflicts:
#	index.html
